app/(server)/_features/user/service.ts

```diff
@@ -0,0 +1,18 @@
+import { type InsertUser } from './schema';
+import { type UserRepo } from './repository';
+
+export class UserService {
+  constructor(private repo: InstanceType<typeof UserRepo>) {}
+
+  async createUser(data: InsertUser) {
+    return this.repo.addUser(data);
+  }
+
+  async getUserById(userId: number) {
+    return this.repo.getUserById(userId);
+  }
+
+  async getUserByEmail(email: string) {
+    return this.repo.getUserByEmail(email);
+  }
+}
```

Comment:
since this service layer doesn't perform any other business logic other than calling the repo function let's kept it simple by not implementing the service layer

Reply:
Sure, I think it's better to remove this service layer.

Reply:
Also, to make importing the repository layer simpler, I think. we don't need to make a class for repository layer. Just exporting the modules would be enough and no need to instantiate the class. Updated on this commit [4e38848](https://github.com/inlivedev/inlive-room/pull/128/commits/4e3884853d4454a387c1d82977cec90bad44f520)